Automattic/pocket-casts-android
# Display Top 1 Show story

## Description

This adds number of shows story.

Figma: lH66LwxxgG8btQ8NrM0ldx-fi-3070_23365

As I commented on Figma I'm missing sticker asset so right now I'm mocking it with rounded rectangle shape. There's also an issue on small devices with text displayed over the cover, which I also asked about.

> [!note]
> Edge to edge is not supported ATM due to the legacy code. Enabling it would break the UI in the whole app. I'll work on supporting it as a separate task.

## Testing Instructions

1. Sign in with an account that listened to at least 5 minutes of episodes this year.
2. Sync your profile data.
3. Close the app.
4. Reopen the app.
5. Open Playback 2024.
6. You might see a loading progress bar and it might take some time to sync your data.
7. Once data is synced go to the top 1 show story and verify it.

## Screenshots or Screencast 

### Preview

| Regular | Small |
| - | - |
| ![big](https://github.com/user-attachments/assets/d45bc0b0-34dc-43bb-9d1a-7efcd4017d56) | ![smol](https://github.com/user-attachments/assets/38b1879c-72ba-4d20-9c6d-590c889d6827) |

### Animation

https://github.com/user-attachments/assets/c47e18ed-b68b-4b90-a5df-8983aaa187b7

## Checklist
 
#### I have tested any UI changes...
